helm/nifi-cluster/templates/nifi-cluster.yaml

```diff
@@ -129,7 +129,7 @@ spec:
         namespace: {{ .Release.Namespace }}
     bootstrapProperties:
       overrideSecretConfig:
-        data: bootstrap.properties
+        data: bootstrap.conf
```

Comment:
I think this change will prompt nifikop to do a rolling restart of the NiFi pods, but this is okay IMO.

Reply:
... maybe. I'm not certain of this. If the filename changes in the Pod's filesystem it will. Otherwise it won't.

Regardless, i'm not concerned.

Reply:
It will, but I'm not concerned either 😄